python/pyspark/streaming/context.py

```diff
@@ -313,14 +331,19 @@ def binaryRecordsStream(self, directory, recordLength):
             self._jssc.binaryRecordsStream(directory, recordLength), self, NoOpSerializer()
         )
 
-    def _check_serializers(self, rdds):
+    def _check_serializers(self, rdds: List["RDD[T]"]) -> None:
```

Comment:
Here, too.

`List["RDD[T]"]` -> `List[RDD[T]]` should be fine.

Reply:
Similar https://github.com/apache/spark/pull/34293#discussion_r787464897, we need inline type hints `python/pyspark/rdd.py` to do that